components/you_need_a_budget/actions/update-transaction/update-transaction.mjs

```diff
@@ -0,0 +1,105 @@
+import app from "../../you_need_a_budget.app.mjs";
+
+export default {
+  key: "you_need_a_budget-update-transaction",
+  name: "Update Transaction",
+  description: "Updates existing transaction. [See the docs](https://api.youneedabudget.com/v1#/Transactions/updateTransaction)",
```

Comment:
```suggestion
  description: "Update an existing transaction. [See the docs](https://api.youneedabudget.com/v1#/Transactions/updateTransaction)",
```

Reply:
Changed "Updates" to "Update" to follow Google's Developer docs style guide (use present tense).